_sources/Unit5-Writing-Classes/topic-5-2-writing-constructors.rst

```diff
@@ -635,7 +641,7 @@ Summary
 
 - **Constructors** are used to set the initial state of an object, which includes initial values for all instance variables.
 
-- When no constructor is written, Java provides a no-argument **default constructor**, and the instance variables are set to their default values (0 for int and double, null for objects like String).
+- When no constructor is written, Java provides a no-argument **default constructor**, and the instance variables are set to their default values (0 for ``int`` and ``double``, null for objects like ``String``).
```

Comment:
```suggestion
- When no constructor is written, Java provides a no-argument **default constructor**, and the instance variables are set to their default values (0 for ``int`` and ``double``, ``null`` for objects like ``String``).
```

Reply:
I'm torn about how to format literals like `0`. Should they be formatted as code? In this case, I'd probably use the word "zero" except that the book as a whole seems to rather aggressively prefer numerals over words.

Reply:
I don't think I would format the literal 0 as code, but either way works. I think I do have a preference for 0 over zero here, but not sure why!